docs/names/howto/listings/names/example-domain.com

```diff
@@ -33,5 +33,6 @@ zone = [
     CNAME('ftp.example-domain.com', 'example-domain.com'),
 
     MX('example-domain.com', 0, 'mail.example-domain.com'),
-    A('mail.example-domain.com', '123.0.16.43')
+    A('mail.example-domain.com', '123.0.16.43'),
+    PTR('43.16.0.123.in-addr.arpa', 'mail.example-domain.com')
```

Comment:
Can you add a trailing comma to this line so that future diffs of this kind won't touch it?

Reply:
Fixed